### DISCUSSION TITLE
Server name - game's browser

### DISCUSSION THREAD

**Discussion by s2b3k** [mbround18/valheim-docker#631] [Q&A]
Hi, maybe I'm blind, but I couldn't find a problem like this by googling...

What can I do to force ingame server browser to show server's name instead of its IP address?

![image](https://user-images.githubusercontent.com/81764702/206475410-3c70cfe9-87a7-4832-b63c-e5667b0ca8aa.png)

I create server on AWS EC2 instances with port forwarding (2450-2480).

**Comment by mbround18** [comment]
[Using NAME sets the name for the server browser](https://github.com/mbround18/valheim-docker#environment-variables) if you add to fav via the IP address it will stay as the ip address

**Reply by s2b3k** [reply]
I set custom NAME, and even if I woudn't default should be used, right?

All the servers from my screenshot were added via IP address. The only one with actual name comes from 4netplayers provider. All of my EC2 servers don't have any detailed info on the list even if they are online.

**Comment by mbround18** [comment]
@s2b3k i was able to solve this! with a bit of DNS magic! :)

![server browser with name](https://user-images.githubusercontent.com/12646562/223612847-a6a9e28d-df72-4669-8e93-c7ca7f102559.png)

![image](https://user-images.githubusercontent.com/12646562/223613082-95627c7a-77ac-4575-a4fb-7b606ea0f053.png)
